pyensign/ensign.py

```diff
@@ -766,7 +768,6 @@ async def wrapper(*args, **kwargs):
                 _client = None
                 raise e
 
-            await _client.close()
```

Comment:
This is redundant because `__aexit__` in the Ensign class closes the client when the code above exits the `async with` block.